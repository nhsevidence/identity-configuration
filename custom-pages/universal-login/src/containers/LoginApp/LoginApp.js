import React from 'react'
import Header from "../../components/Header";
import Nav from "../../components/Navigation";
import {Route, Switch} from "react-router";
import Login from "../../routes/Login";
import ForgotPassword from "../../routes/ForgotPassword";
import ForgotPasswordSuccess from "../../routes/ForgotPasswordSuccess";
import Register from "../../routes/Register";
import RegisterSuccess from "../../routes/RegisterSuccess";
import NotFound from "../../routes/NotFound";
import {Link} from "react-router-dom";
import './LoginApp.scss'

class LoginApp extends React.Component {
  render() {
    return (
      <div>
        <div className="wrapper">
          <Header className="col leftCol" />
          <div className="mainContainer col rightCol">
            <Switch>
              <Route exact path="/" component={Login.component} />
              <Route exact path='/forgotPassword' component={ForgotPassword.component} />
              <Route exact path='/forgotsuccess' component={ForgotPasswordSuccess.component} />
              <Route exact path='/register' component={Register.component} />
              <Route exact path='/regsuccess' component={RegisterSuccess.component} />
              <Route component={NotFound.component} />
            </Switch>
            <Link className="forgotPasswordLink" data-qa-sel="forgotPassword-link" to="/forgotPassword">
              Forgot password?
            </Link>
          </div>
        </div>
      </div>
    )
  }
}
export default LoginApp
