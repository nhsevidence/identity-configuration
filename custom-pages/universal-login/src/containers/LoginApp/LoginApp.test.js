import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import LoginApp from './LoginApp'

Enzyme.configure({ adapter: new Adapter() });

const props = { history: {}, routes: {}, routerKey: 1 };

describe('LoginApp components', () => {
  it('should render <LoginApp /> correctly', () => {
    expect(shallow(<LoginApp {...props} />)).toMatchSnapshot()
  })
});
