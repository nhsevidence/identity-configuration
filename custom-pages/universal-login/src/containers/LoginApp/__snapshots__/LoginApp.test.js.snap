// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginApp components should render <LoginApp /> correctly 1`] = `ShallowWrapper {}`;
